### DISCUSSION TITLE
屏蔽3分钟以下短视频

### DISCUSSION THREAD

**Discussion by Lambert-Rao** [the1812/Bilibili-Evolved#4680] [功能建议 / Ideas]
各位好，因为b站有很多低质量视频是短视频（不超过1分钟），不知道是否已经有实现屏蔽n(自定义) 分钟以下视频的功能？
搜索页面可以调整10-30分钟，但是这样还是不能满足使用需求，会屏蔽掉5-10分钟的优质视频。
除了搜索页面，更主要的是希望能在首页推荐上去掉这些低质量短视频。不知道有无实现方法？
如果我提idea的方式不对，还请各位包涵并指出，这是我第一次提出discussion :)

**Comment by hexadecimal233** [comment]
提供一个思路，那种评论少，点赞率不高的基本上可以确定是低质量的了，可以考虑获取评论数和点赞数去屏蔽

**Comment by Geeyun-JY3** [comment]
重复，请善用搜索：
- 搜索页面：#2997、#3622
- 首页推荐：<https://github.com/the1812/Bilibili-Evolved/discussions/4289#discussioncomment-6530265>、#4637

另外昨天刚说的不会做的功能：<https://github.com/the1812/Bilibili-Evolved/discussions/1301#discussioncomment-8787140>，含「屏蔽 / 筛选满足某些条件的视频」。

**Comment by Lambert-Rao** [comment]
thank you for your reply
sorry for troubling you
